### PR TITLE
replaced broken link

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8809

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Once activated, Microsoft Defender for Endpoint will immediately start forwardin
 
 ## View the data collected
 
-To view and access Microsoft Defender for Endpoint data in Microsoft Cloud Apps Security, see [Investigate devices in Cloud App Security](https://docs.microsoft.com/cloud-app-security/wdatp-integration#investigate-machines-in-cloud-app-security).
+To view and access Microsoft Defender for Endpoint data in Microsoft Cloud Apps Security, see [Investigate devices in Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/mde-integration#investigate-devices-in-cloud-app-security).
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md:47]: This link also works in a locale-agnostic format: https://docs.microsoft.com/cloud-app-security/mde-integration#investigate-devices-in-cloud-app-security
```suggestion
To view and access Microsoft Defender for Endpoint data in Microsoft Cloud Apps Security, see [Investigate devices in Cloud App Security](https://docs.microsoft.com/cloud-app-security/mde-integration#investigate-devices-in-cloud-app-security).
```
BTW, thanks for finding the change from "wsdatp" to "mde" as part of the new name for Defender links.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Please remove the locale from the link as suggested by @illfated.

**Review by denisebmsft — APPROVED** [review]
Thank you @MaratMussabekov @JohanFreelancer9 @illfated

**Pull request merged**